torchci/pages/cost_analysis.tsx

```diff
@@ -0,0 +1,648 @@
+import {
+  Box,
+  FormControl,
+  FormGroup,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+  Stack,
+  TextField,
+  ToggleButton,
+  ToggleButtonGroup,
+  Tooltip,
+  Typography,
+} from "@mui/material";
+import { DateTimePicker, LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import TimeSeriesPanel, {
+  ChartType,
+  Granularity,
+} from "components/metrics/panels/TimeSeriesPanel";
+import MultiSelectPicker from "components/MultiSelectPicker";
+import dayjs from "dayjs";
+import { fetcher } from "lib/GeneralUtils";
+// import an image from the public folder, bar-chart.png
+import Image from "next/image";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import { FaFilter, FaInfoCircle } from "react-icons/fa";
+import useSWR from "swr";
+
+function TimePicker({ label, value, setValue }: any) {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DateTimePicker
+        renderInput={(props) => <TextField {...props} />}
+        label={label}
+        value={value}
+        onChange={(newValue) => {
+          setValue(newValue);
+        }}
+      />
+    </LocalizationProvider>
+  );
+}
+
+/**
+ * Allows the user to pick from common time ranges, or manually set their own.
+ */
+export function TimeRangePicker({
```

Comment:
Is this different from the component in metrics.tsx?

Reply:
I actually intended it to be a DatePicker only, and remove the time component. Changed the naming now everywhere to reflect this.